MediaBrowser.Providers/Tmdb/TV/TmdbSeasonImageProvider.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Net;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Dto;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Globalization;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Providers;
+using MediaBrowser.Model.Serialization;
+using MediaBrowser.Providers.Tmdb.Models.General;
+using MediaBrowser.Providers.Tmdb.Movies;
+
+namespace MediaBrowser.Providers.Tmdb.TV
+{
+    public class TmdbSeasonImageProvider : IRemoteImageProvider, IHasOrder
+    {
+        private readonly IJsonSerializer _jsonSerializer;
+        private readonly IHttpClient _httpClient;
+        private readonly IFileSystem _fileSystem;
+        private readonly ILocalizationManager _localization;
+
+        public TmdbSeasonImageProvider(IJsonSerializer jsonSerializer, IHttpClient httpClient, IFileSystem fileSystem, ILocalizationManager localization)
+        {
+            _jsonSerializer = jsonSerializer;
+            _httpClient = httpClient;
+            _fileSystem = fileSystem;
+            _localization = localization;
+        }
+
+        public int Order => 1;
+
+        public string Name => ProviderName;
+
+        public static string ProviderName => TmdbUtils.ProviderName;
+
+        public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken cancellationToken)
+        {
+            return _httpClient.GetResponse(new HttpRequestOptions
+            {
+                CancellationToken = cancellationToken,
+                Url = url
+            });
+        }
+
+        public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, CancellationToken cancellationToken)
+        {
+            var season = (Season)item;
+            var series = season.Series;
+
+            var seriesId = series != null ? series.GetProviderId(MetadataProviders.Tmdb) : null;
```

Comment:
```suggestion
            var seriesId = series ?? series.GetProviderId(MetadataProviders.Tmdb);
```

Reply:
The ?? is new to me so I looked it up in the [docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator). It doesn't seem right to me: 

> The null-coalescing operator ?? returns the value of its left-hand operand if it isn't null; otherwise, it evaluates the right-hand operand and returns its result.

So if series isn't null, we essentially run `var seriesId = series`?

Reply:
Woops my bad, should have been this
```suggestion
            var seriesId = series?.GetProviderId(MetadataProviders.Tmdb);
```
Sorry for all my mistakes